platform/android/java/editor/build.gradle

```diff
@@ -12,6 +12,25 @@ dependencies {
     implementation "androidx.window:window:1.0.0"
 }
 
+ext {
+    // Build number added as a suffix to the version code, and incremented for each build/upload to
+    // the Google Play store.
+    // This should be reset on each stable release of Godot.
+    editorBuildNumber = 0
+    // Value by which the Godot version code should be offset by to make room for the build number
+    editorBuildNumberOffset = 1000
+}
+
+def generateVersionCode() {
+    int libraryVersionCode = getGodotLibraryVersionCode()
+    return (libraryVersionCode * editorBuildNumberOffset) + editorBuildNumber
```

Comment:
To be sure, this is used as a signed 64-bit int? Because we're getting close to `INT32_MAX` already with the numbers chosen (`(4 * 100000) * 1000` is 400,000,000, `INT32_MAX` is `2,100,000,000` and peanuts).

Reply:
You're right, according to the [doc](https://developer.android.com/studio/publish/versioning#appversioning):
> The greatest value Google Play allows for versionCode is 2100000000.

I'll reduce the range for the build number.